### PR TITLE
Expose creation and modification date for subscription

### PR DESCRIPTION
**Pull request by jewertow**
#1123 

For now I implemented it only for subscription. Please check it out and give me a feedback, then I will continue working.

What format of creation and modification date do you expect? Timestamp (`Long`) or human-readable string ("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")?

### CODE REVIEW

File: hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperBasedRepository.java [hunk 1]
```diff
@@ -109,6 +114,31 @@ protected boolean pathExists(String path) {
...
 
+    private <T> Optional<T> readWithStatFrom(String path, ThrowingReader<T> supplier, BiConsumer<T, Stat> statDecorator, boolean quiet) {
+        try {
+            Stat stat = new Stat();
+            byte[] data = zookeeper.getData().storingStatIn(stat).forPath(path);
+            if (ArrayUtils.isNotEmpty(data)) {
+                T t = supplier.read(data);
+                statDecorator.accept(t, stat);
+                return Optional.of(t);
+            } else {
```
Comment (cristaloleg) [inline hermes-common/src/main/java/pl/allegro/tech/hermes/infrastructure/zookeeper/ZookeeperBasedRepository.java]: `else` isn't needed :)

**Comment by jewertow** [comment]
@cristaloleg what date format do you expect?
Default date format:
timestamp1
or date as string:
timestamp2

**Comment by cristaloleg** [comment]
For me 2nd option is more preferable, 'cause it's mostly dev-oriented console, but let's ping @dankraw @druminski @piotrrzysko @faderskd  😉

**Comment by jewertow** [comment]
@cristaloleg I pushed changes in hermes-console. I think PR is ready to review.

**Pull request merged**